python/pyspark/sql/window.py

```diff
@@ -217,11 +242,16 @@ def rangeBetween(self, start, end):
         :param end: boundary end, inclusive.
                     The frame is unbounded if this is ``Window.unboundedFollowing``, or
```

Comment:
Can we rewrite to reflect the current change here? For example, something like .. 

```
``Window.unboundedFollowing`` or a column returned by ``pyspark.sql.functions.unboundedFollowing``?
```